translator/sample files/cotrip_data/cdot-rtdh-prod_cotrip-events-standard-oim-wzdx-integration_20210316-232928-207252.json

```diff
@@ -0,0 +1,36 @@
+{

```

Comment:
There's several of these files that seem to be named with a date, is this something we're using? If so, can we add description to the README and if not, can we remove?

Reply:
removed all those unnecessary files